Steps/UploadCodeCoverage.yml

```diff
@@ -109,13 +143,14 @@ steps:
       UPLOAD_CMD = os.environ['UPLOAD_CMD']
 
       for cov_file in Path(REPORT_DIR).rglob('*coverage.xml'):
-        params = f'{UPLOAD_CMD} -f {cov_file} -Z'
+        cmd = f'{UPLOAD_CMD} -f {cov_file} -Z'
         if COV_FLAG:
-          params += f' -F {COV_FLAG}'
+          cmd += f' -F {COV_FLAG}'
         process = subprocess.Popen(
-          [UPLOAD_CMD, params],
+          cmd,
           stdout=subprocess.PIPE,
-          stderr=subprocess.PIPE)
+          stderr=subprocess.PIPE,
+          shell=True)
```

Comment:
Generally, you should avoid using the shell. Was that needed? I thought everything would be resolved going into the final string.

Reply:
Nvm, I think you're depending on that to implicitly get the token.

Reply:
The problem (openness) and simplicity of env vars.